examples/01_example_wsiread.ipynb

```diff
@@ -12,7 +12,7 @@
     "\n",
```

Comment:
<p>The text cell above should be deleted, and the code cell below should be deleted.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TIA-Lab/tiatoolbox/pull/132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Addressed in this commit https://github.com/TIA-Lab/tiatoolbox/pull/132/commits/f63e9bef28deb14b8e74a62bb9df12806ebceedc